README.md

```diff
@@ -1,6 +1,6 @@
 # BankApi
 
-BankApi wraps chilean banks web operations.
+BankApi is a gem that wraps chilean banks web operations. For now, it can get recent deposits on Banco de Chile accounts.
```

Comment:
le agregaría enterprise: "it can get recent deposits on Banco de Chile enterprise accounts"

Reply:
Filo, lo dices más abajo